src/main/java/bitflow4j/io/file/OutputFiles.java

```diff
@@ -32,18 +34,25 @@
     private final String fileNameTemplate;
 
     public OutputFiles(String file) {
-        this(file, EndpointFactory.guessFormat(Endpoint.Type.FILE, file).getMarshaller());
+        this(file, (String) null);
```

Comment:
Seit wann muss man null zu String parsen 💃

Reply:
Der Grund ist: wenn man einfach nur **null** in diesem this(...) Konstruktor gibt, ist nicht eindeutig klar, welcher Konstruktor gemeint ist, weil es mehrere gibt, wo **null** als Wert "reinpassen" würde. Man muss es explizit nach String casten, um klar zu machen, welcher Konstruktor gemeint ist.